### PR TITLE
Documentation issue form update

### PR DESCRIPTION
**Pull request by NivekT**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #215

Merge this after the website is live.

Differential Revision: [D34478899](https://our.internmc.facebook.com/intern/diff/D34478899)

### CODE REVIEW

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34478899).

File: .github/ISSUE_TEMPLATE/documentation.yml [hunk 1]
```diff
@@ -6,11 +6,10 @@ body:
...
       description: >
-        A clear and concise description of what content in within the inline documentation has an issue. We should have
-        additional documentation on the PyTorch website soon. If this has to do with the general https://pytorch.org
-        website, please file an issue at https://github.com/pytorch/pytorch.github.io/issues/new/choose instead. If this
-        has to do with https://pytorch.org/tutorials, please file an issue at
-        https://github.com/pytorch/tutorials/issues/new.
+        A clear and concise description of what content in https://pytorch.org/data is an issue. If this has to do with
+        the general https://pytorch.org website, please file an issue at
+        https://github.com/pytorch/pytorch.github.io/issues/new/choose instead. If this has to do with
+        https://pytorch.org/tutorials, please file an issue at https://github.com/pytorch/tutorials/issues/new.
```
Comment (ejguan) [inline .github/ISSUE_TEMPLATE/documentation.yml:12]: As we are providing tutorial on https://pytorch.org/data/main/tutorial.html, we should still let users to file issue in TorchData.

**Review by ejguan — APPROVED** [review]
I think we can land it now as the website has already been alive

**Comment by NivekT** [comment]
@nivekt has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D34478899).

**Pull request closed without merging**